### PR TITLE
Abb 207

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Abb 207 new (description in catalogue scatchy), minor changes in others.

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]
ok just resolve conflict

File: ParisBNF/abb/BNFabb207.xml [hunk 1]
```diff
@@ -0,0 +1,278 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb207" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Book of the Perl, Malkǝʾ-hymn to Our Lady Mary, protective texts</title>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb207.xml:7]: Pearl

File: ParisBNF/abb/BNFabb207.xml [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+                              <incipit xml:lang="gez">በስመ፡ አብ፡ <gap reason="ellipsis" resp="CR"/>ዛሙስ፡ የአድ፡ ታውሚ፡ አስሆሩ<gap reason="illegible" unit="chars" quantity="1" resp="CR"/>ብለባርያ፡ 
+                                  አገባጅ፡ ዘበላዕለ፡ ምሻቱር፡ ሰሎሞን፡ አርምኃ። <gap reason="ellipsis" resp="CR"/>ወለአጋንንት፡ ለደስክ፡ ወለጉዳሌ፡ ለቡዳ፡ ወለተግባረ፡ ሰብእ፡ ለውግዓት፡ 
+                                  ወለቁርጥማ፡ ለድድቅ፡ ወለጋኔነ፡ ቀትር፡ ለፌራ፡ ወለዳስማ፡ ለምትአት፡ ወለፅፋፍአት፡</incipit>
+                         </msItem>
+
+                                <msItem xml:id="p1_i4">
+                            <locus from="38r"/>
+                                    <title type="incomplete" ref="LIT7022HemamaBedbed">ጸሎት፡ በእንተ፡ ሕማመ፡ ብድብድ፡</title>
+                             <textLang mainLang="gez"/>
+                                    <note>How the Lord spoke to ʾAmru at Mirod, he sent to ʾAmru an angel called Ṭǝqonṭoqyā,
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb207.xml:132]: Is it better to indicate, that this is a translation of a portion of the text? E.g. 
`'How the Lord spoke to ʾAmru at Mirod, he sent to ʾAmru an angel called Ṭǝqonṭoqyā, (...)'`

File: ParisBNF/abb/BNFabb207.xml [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+                                </supportDesc>
+                                <layoutDesc>
+                                    <layout columns="2" writtenLines="10"/> 
+                                </layoutDesc>                       
+                                </objectDesc>
+                                
+                                <handDesc>
+                                    <handNote xml:id="h1" script="Ethiopic" corresp="#p1 #p2_i2">                                        
+                                        <seg type="script">18th-century script.</seg>                                        
+                                    </handNote>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb207.xml:156]: Is it useful to provide a date in `<date notBefore="1700" notAfter="1799" resp="DN"/>` in addition to
 `<seg type="script">18th-century script.</seg>` or is it redundant?

File: ParisBNF/abb/BNFabb207.xml [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+                                  <locus from="45r" to="54v"/>                                  
+                                </layout>
+                             </layoutDesc>
+                        </objectDesc>
+
+                        <handDesc>
+                           <handNote xml:id="h2" script="Ethiopic">
+                              <seg type="script">18th-century script</seg>
+                              <seg type="ink">Black, red</seg>
+                               <desc>The script of extremely small size.</desc>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb207.xml:216]: I suggest either `The script is of extremely small size.` or `Script of extremely small size.`

File: ParisBNF/abb/BNFabb207.xml [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+                              </handNote>
+                        </handDesc>
+                        
+                        <bindingDesc>
+                           <binding contemporary="true" xml:id="binding">
+                              <decoNote xml:id="b2">The quire it is infixed into <ref target="#p1">Unit 1</ref>
+                                  </decoNote>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb207.xml:223]: Probably better: `The quire is inserted into <ref target="#p1">Unit 1</ref>.` or `The quire it is attached to <ref target="#p1">Unit 1</ref>.` (In any case without 'it', but with a period.)

File: ParisBNF/abb/BNFabb207.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb207.xml:7]: Ok. (In another branch)

File: ParisBNF/abb/BNFabb207.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb207.xml:132]: This is how I styled the renarration of Conti Rossini.

File: ParisBNF/abb/BNFabb207.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb207.xml:156]: I used to shape this segment in this way; here it is simple, but most frequently, concerning the script, we must sway away from exact numbers (if we say "late", "close to the end", "early", "around the middle"). If you think it is importan, please open an issue.

File: ParisBNF/abb/BNFabb207.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb207.xml:216]: ok

File: ParisBNF/abb/BNFabb207.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb207.xml:223]: I corrected, "infix" = "einfügen", though, slightly different from "attach".

File: ParisBNF/abb/BNFabb207.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply ParisBNF/abb/BNFabb207.xml:156]: I just wanted to know, whether <date/> is neccessary to make the hands searchable. I will open an issue to clarify this question.

**Pull request merged**